src/kibit/driver.clj

```diff
@@ -39,10 +39,10 @@
                                         :reporter (name-to-reporter (:reporter options)
                                                                     cli-reporter)
                                         :rules (or rules all-rules))
-                            (catch Exception e
+                            (catch Throwable e
                               (binding [*out* *err*]
-                                (println "Check failed -- skipping rest of file")
-                                (println (.getMessage e))))))
+                                (println (str "At " (.getPath file) ":0:\nCheck failed -- skipping rest of file"))
```

Comment:
is it possible to get the real line number here? If we just log the stack trace, can Emacs understand that?